document process and current members (#19)
haskell/security-advisories: README.md

The above [TOML] "front matter" is followed by the long description in [Markdown] format.

## Current Members

- [Tristan de Cacqueray](tristan.cacqueray@gmail.com)

> - [Gautier di Folco](mailto:gautier.difolco@gmail.com)

Reviewer: ```suggestion
- [Gautier Di Folco](mailto:gautier.difolco@gmail.com)
```

- [Mihai Maruseac](mailto:mihai.maruseac@gmail.com)
- [Casey Mattingly](mailto:case@capsulecorp.org)
- [David Christiansen Thrane](david@haskell.foundation)
- [Fraser Tweedale](frase@frase.id.au)

## Processes

Haskell Security Advisory Database is a for security vulnerabilities. Here are some examples:

> - RCE

Reviewer: ```suggestion
- Remote Code Execution
```

- Memory Corruption
- Privilege Escalation
- Cryptography issues
- Various injection issues (SQL, Command, etc.)
- Use of broken algorithms
- Use of obsolete libraries

When in doubt, submit a PR. As a group, we will review and decide to promote the vulnerability in the Database.
If there are issues regarding confidentiality and you would like to disclose an issue in a more private setting, please use the [Mailing List](mailto:security-advisories@haskell.org)

As a group, we will review the submission and collectively decide whether or not we'll issue an advisory.

## Acknowledgments

> The process and documentation in this repository are based off the work of the [RustSec][RustSec] team.

Reviewer: ```suggestion
The process and documentation in this repository are based off the work of the [RustSec](https://rustsec.org/) team.
```

The above [TOML] "front matter" is followed by the long description in [Markdown] format.

## Current Members

- [Tristan de Cacqueray](mailto:tristan.cacqueray@gmail.com)
- [Gautier Di Folco](mailto:gautier.difolco@gmail.com)
- [Mihai Maruseac](mailto:mihai.maruseac@gmail.com)
- [Casey Mattingly](mailto:case@capsulecorp.org)
- [David Thrane Christiansen](mailto:david@haskell.foundation)
- [Fraser Tweedale](mailto:frase@frase.id.au)

## Processes

> please see [Contributing](./CONTRIBUTING.md) for details.

Reviewer 2: ```suggestion
Please see [Contributing](./CONTRIBUTING.md) for details.
```

Sorry to be a stickler :-)

The above [TOML] "front matter" is followed by the long description in [Markdown] format.

## Current Members

- [Tristan de Cacqueray](tristan.cacqueray@gmail.com)

> - [Gautier Di Folco](mailto:gautier.difolco@gmail.com)

Reviewer: that's fine

- [Mihai Maruseac](mailto:mihai.maruseac@gmail.com)
- [Casey Mattingly](mailto:case@capsulecorp.org)
- [David Christiansen Thrane](david@haskell.foundation)
- [Fraser Tweedale](frase@frase.id.au)

## Processes

> Haskell Security Advisory Database is a for security vulnerabilities. Here are some examples:

Reviewer 2: ```suggestion
The Haskell Security Advisory Database is a collection of security advisories for Haskell libraries. Here are some examples of the kinds of issues that are documented in the database:
```

- Remote Code Execution
- Memory Corruption
- Privilege Escalation
- Cryptography issues
- Various injection issues (SQL, Command, etc.)
- Use of broken algorithms
- Use of obsolete libraries

When in doubt, submit a PR. As a group, we will review and decide to promote the vulnerability in the Database.

> If there are issues regarding confidentiality and you would like to disclose an issue in a more private setting, please use the [Mailing List](mailto:security-advisories@haskell.org)

Reviewer 2: ```suggestion
If there are issues regarding confidentiality and you would like to disclose an issue in a more private setting, please use the [mailing list](mailto:security-advisories@haskell.org). It reaches only the members of the committee, and the archives are private.
```
